iterators task (1)/student/include/impl.h

```diff
@@ -5,11 +5,47 @@ template <class Container>
 struct accumulator
 {
     /** @todo Iterator traits*/
+    using ContainerTraits = std::iterator_traits<typename Container::iterator>;
```

Comment:
Nice solution, but does it has sense for output iterator?
Usually, for output iterators, properties are defined as void